servers/dsub/jobs/controllers/utils/time_frame.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+from dateutil.tz import tzlocal
+
+TIME_FRAME_MAP = {
+    'HOURS_1': datetime.timedelta(minutes=60),
+    'HOURS_8': datetime.timedelta(hours=8),
+    'HOURS_24': datetime.timedelta(days=1),
+    'DAYS_7': datetime.timedelta(days=7),
+    'DAYS_30': datetime.timedelta(days=30)
+}
+
+
+def time_frame_to_start_time(time_frame):
```

Comment:
tiny nit: I would just call this `start_time` or `to_start_time` since the caller is doing `timeframe.time_frame_to_start_time` right now the extract "time_frame" seems unnecessary